.releaserc.yml

```diff
@@ -38,6 +38,21 @@ plugins:
               hasChanged: true
               numMatches: 1
               numReplacements: 1
+        - files:
+            - rust/cosmos_sdk_grpc_client/Cargo.toml
+            - rust/okp4_grpc_client/Cargo.toml
```

Comment:
We also need to reference those files in the assets of the `@semantic-release/git` plugin, otherwise they won't be committed.